keras/engine/topology.py

```diff
@@ -1229,6 +1229,45 @@ def count_params(self):
                                    self.name + '.build(batch_input_shape)`.')
         return sum([K.count_params(p) for p in self.weights])
 
+    def count_flops(self):
```

Comment:
I also don't think the confusion with TF ops is a big deal. As long as the docstring makes it very clear what is being counted.

One more note: I don't think op counting is a common enough use case that it should be displayed in `model.summary()`.